x/emissions/module/rewards/rewards.go

```diff
@@ -513,8 +513,20 @@ func payoutRewards(
 
 			if reward.Type == types.WorkerInferenceRewardType {
 				infererRewards = append(infererRewards, reward)
+
```

Comment:
Might be worth writing tests for these changes

Reply:
Narrator: he wrote exactly one test, and it wasn't very thorough